sonic-psud/scripts/psud

```diff
@@ -205,10 +205,16 @@ class PsuChassisInfo(logger.Logger):
         for index, psu in enumerate(self.chassis.get_all_psus()):
             presence = try_get(psu.get_presence)
             if not presence:
+                name = try_get(psu.get_name, 'PSU {}'.format(index + 1))
+                chassis_tbl.hdel(CHASSIS_INFO_POWER_KEY_TEMPLATE.format(1), CHASSIS_INFO_POWER_SUPPLIER_FIELD.format(name))
                 continue
 
             power_good = try_get(psu.get_powergood_status)
             if not power_good:
+                name = try_get(psu.get_name, 'PSU {}'.format(index + 1))
```

Comment:
can we retrieve the `name` outside instead of repeating in the same block?

Reply:
@prgeor,  Hi Prince, This comment has been addressed. Please review it. Thanks